package.json

```diff
@@ -27,18 +27,16 @@
     "brunch": "./bin/brunch"
   },
   "scripts": {
-    "prepublish": "node setup.js prepublish",
-    "postpublish": "node setup.js postpublish",
-    "test": "node setup.js test",
-    "postinstall": "node setup.js postinstall"
+    "prepublish": "coffee -o lib/ src/",
```

Comment:
i'm not sure whether this is a cross-platform solution. can someone check for win32 support?